src/atoms/visual-story/cover-page/cover-page.tsx

```diff
@@ -124,6 +130,8 @@ const StyledHeadline = styled.h1`
   line-height: ${(props) => props.theme.font.lineHeight.level2};
   font-weight: ${(props) => props.theme.font.weight.medium};
   font-size: ${(props) => props.theme.font.size.huge};
+  font-family: ${(props) => props.theme.font.family.primary};
+  word-break: break-word;
```

Comment:
@shraddha-kesari `break-word` is deprecated. Can we use an alternative for this

Reply:
Its not https://caniuse.com/?search=break-word